test/bigunit.operations.test.ts

```diff
@@ -195,13 +195,14 @@ describe("BigUnit Class Methods", () => {
       describe("multiply", () => {
         const mulRespectingPrecisionOverride = [
           // num1, precision1, num2, precision2, overrideprecision, expectedString
+          // two numbers with the same precision
           [100.1, 5, 1.23456, 5, 1, "120.12000"], // = 100.10000 * 1.2
-          // different precission between 2 BigUnits
+          // two numbers with the different precision
           [1.23456, 5, 10.999, 3, 2, "13.56781"], // = 1.23456 * 10.99
-          // zero multiply a number
-          [0, 9, 0.5, 4, 3, "0.000000000"], // = 1.001001001 * 0.565
-          // has a negative number
-          [1.00010001, 9, -0.5656, 4, 10, "-0.5656565656"], // = 1.00010001 * 0.5656
+          // zero multiplied by a number
+          [0, 9, 0.5, 4, 3, "0.000000000"], // = 0.000000000 * 0.500
+          // has a negative number and override precision is larger than either inputs precision
+          [1.00010001, 9, -0.5656, 4, 10, "-0.5656565656"], // = 1.000100010 * 0.5656000000
         ];
         test.each(mulRespectingPrecisionOverride)(
           "rodo: should multiply two BigUnit instances, number %f with precision %i and number %f with precision %i, respecting a precision override of %i and result in %s",
```

Comment:
Why is Rodo's name in these files

Reply:
sorry, I forgot to delete it in the previous PR...